### PR TITLE
Added script to fetch wheels [ci skip]

### PR DESCRIPTION
**Pull request by TomAugspurger**
[ci skip]

### CODE REVIEW

**Comment by TomAugspurger** [comment]
Tyring to automate as much of the release process as possible.

**Comment by TomAugspurger** [comment]
Merging later today if there are no objections.

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jreback** [comment]
thanks @TomAugspurger

File: scripts/download_wheels.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+"""Fetch wheels from wheels.scipy.org for a pandas version."""
+import argparse
```
Comment (jreback) [inline scripts/download_wheels.py]: might need

```
#!/usr/bin/env python
# -*- coding: utf-8 -*-
```

File: scripts/download_wheels.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+def fetch(version):
+    base = 'http://wheels.scipy.org'
+    tree = html.parse(base)
+    root = tree.getroot()
+
+    dest = pathlib.Path('dist')
+    dest.mkdir(exist_ok=True)
+
+    files = [x for x in root.xpath("//a/text()")
+             if x.startswith(f'pandas-{version}')
```
Comment (jreback) [inline scripts/download_wheels.py]: this only works on >= 3.6 (which is ok), but the linter is running this.

**Pull request merged**